ehitchhiking-ui/src/app/shared/components/active-trips-modal/active-trips-modal.component.ts

```diff
@@ -0,0 +1,85 @@
+import { Component, ElementRef, OnInit, ViewChild } from "@angular/core";
+import { LoaderSize } from "../../enums/pre-loader-sizes";
+import { MatDialogRef } from "@angular/material";
+import { ActiveTripsModalService } from "./active-trips-modal.service";
+import { Trip } from "../trips-modal/trips";
+import { ActiveTrip } from "../active-trip/active-trip";
+
+
+@Component({
+  selector: 'app-active-trips-modal',
+  templateUrl: './active-trips-modal.component.html',
+  styleUrls: ['./active-trips-modal.component.sass'],
+  providers: [ActiveTripsModalService],
+})
+export class ActiveTripsModalComponent implements OnInit {
+
+  limit = 5;
```

Comment:
Why not 4? Pls use the same constant as in the "History" modal

Reply:
limit =4